### ISSUE TITLE
Cannot use properly kubernetes package

### ISSUE THREAD

**Issue by anquegi** [hlissner/doom-emacs#5246]
### What did you expect to happen?

First of all thanks for doing doom-emacs, finally I really enjoy emacs
After configure the package [kubernetes](https://github.com/kubernetes-el/kubernetes-el), I followed the doc and this issue https://github.com/hlissner/doom-emacs/issues/1700. I cannot set the variables kubernetes-poll-frequency and kubernetes-redraw-frequency. This is my setup:

packages.el:

```
(package! kubernetes)

;; If you want to pull in the Evil compatibility package.
(package! kubernetes-evil)

```

config.el:

```
(use-package! kubernetes
  :commands (kubernetes-overview))

(after! kubernetes
  (setq! kubernetes-poll-frequency 3600
         kubernetes-redraw-frequency 3600))

;; If you want to pull in the Evil compatibility package.
(use-package! kubernetes-evil
  :after kubernetes)

```

The package works quite well, but hags so I want to set this variables as they recommend in the README

<img width="1680" alt="Captura de pantalla 2021-07-09 a las 16 29 06" src="https://user-images.githubusercontent.com/2518346/125094330-74f70880-e0d3-11eb-8a4e-048115a5ee1f.png">

```
ELISP> kubernetes-poll-frequency
5 (#o5, #x5, ?\C-e)
ELISP> kubernetes-redraw-frequency
5 (#o5, #x5, ?\C-e)
```

So I'm not sure how to configure, maybe the problem is taht I'm using emacs 28 native compilation, and also I get an error when I try the command

### What actually happened?

But the changes doesn't apply

```
ELISP> kubernetes-poll-frequency
5 (#o5, #x5, ?\C-e)
ELISP> kubernetes-redraw-frequency
5 (#o5, #x5, ?\C-e)
```

So I'm not sure how to configure, maybe the problem is taht I'm using emacs 28 native compilation, and also I get an error when I try the command:

```
Error (kubernetes-redraw-hook): Error running hook "kubernetes-overview--redraw-buffer" because: (wrong-type-argument (or eieio-object class) nil obj) Disable showing Disable logging
```

### Describe your attempts to resolve the issue

I changed from the actual configuration from this:

```
(use-package! kubernetes
  :commands (kubernetes-overview)
  :config
  (setq kubernetes-poll-frequency 3600
        kubernetes-redraw-frequency 3600))
```

set this variables in a ielm session

### Steps to reproduce

Basically install this packages, doom sync and start doom emacs in my case as a daemon

### System Information

https://pastebin.com/aFSwJcTn

**Comment by hlissner** [comment]
I added

```elisp
;; to ~/.doom.d/config.el
(use-package! kubernetes
  :commands (kubernetes-overview)
  :config
  (setq kubernetes-poll-frequency 3600
        kubernetes-redraw-frequency 3600))

;; to ~/.doom.d/packages.el
(package! kubernetes)
(package! kubernetes-evil)
```

Ran `doom sync`, restarrted Emacs, and loaded `kubernetes` and `kubernetes-{poll,redraw}-frequency` were successfully changed, so this is likely a local issue and not a Doom issue.

Make sure there are no other errors in your config (e.g. at startup or when loading `kubernetes` for the first time). Check \*Messages\* for warnings/error messages (`C-h e` to open this buffer).

I'll close this until I have reason to believe it is a Doom issue.
